cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -416,6 +405,20 @@ func validateNodeServerCert(ctx *controllerContext, csr *capi.CertificateSigning
 	return false, nil
 }
 
+func getInstanceIps(ifaces []*compute.NetworkInterface) []string {
+	var ips []string
+	for _, iface := range ifaces {
+		ips = append(ips, iface.NetworkIP, iface.Ipv6Address)
```

Comment:
do we need to check if Ipv6Address, NatIP, or ExternalIpv6 are empty before appending?

Reply:
Yeah. I think we should do that. Having "" entries in the slice won't result in incorrect behaviour but is wasteful.